hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -2429,15 +2431,11 @@ public FlushResultImpl flushcache(List<byte[]> families,
         if (!writestate.flushing && writestate.writesEnabled) {
           this.writestate.flushing = true;
         } else {
-          if (LOG.isDebugEnabled()) {
-            LOG.debug("NOT flushing memstore for region " + this
-                + ", flushing=" + writestate.flushing + ", writesEnabled="
-                + writestate.writesEnabled);
-          }
-          String msg = "Not flushing since "
-              + (writestate.flushing ? "already flushing"
-              : "writes not enabled");
+          String msg = "NOT flushing " + this + "; " +
```

Comment:
The log message was being constructed anyways, whether debug or not to put into the status.abort. The if debug was sort of silly.

 @@ -2429,15 +2431,11 @@ public class HRegion implements HeapSize, PropagatingConfigurationObserver, Regi
          if (!writestate.flushing && writestate.writesEnabled) {
            this.writestate.flushing = true;
          } else {
 -          if (LOG.isDebugEnabled()) {
 -            LOG.debug("NOT flushing memstore for region " + this
 -                + ", flushing=" + writestate.flushing + ", writesEnabled="
 -                + writestate.writesEnabled);
 -          }
 -          String msg = "Not flushing since "
 -              + (writestate.flushing ? "already flushing"
 -              : "writes not enabled");
 +          String msg = "NOT flushing " + this + "; " +
 +            (writestate.flushing ? "already flushing": "writes are not enabled");
 +          LOG.debug(msg);
            status.abort(msg);